sdk/monitor/monitor-opentelemetry/package.json

```diff
@@ -19,7 +19,7 @@
     "generate:client": "autorest --typescript ./swagger/README.md",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "integration-test:browser": "echo skipped",
-    "integration-test:node": "dev-tool run test:node-ts-input --no-test-proxy -- \"test/internal/functional/**/*.test.ts\"",
+    "integration-test:node": "dev-tool run test:vitest --esm --no-test-proxy -- \"test/internal/functional/*.test.ts\"",
```

Comment:
Shouldn't we just rename the files to spec?

Reply:
I will do that on a separate PR